Separate QGIS desktop and server documentation (#5930)
qgis/QGIS-Documentation: docs/user_manual/preamble/features.rst

Publish maps on the Internet
----------------------------------------------------------------------

QGIS can be used as a WMS, WMTS, WMS-C or WFS and WFS-T client (see
section :ref:`working_with_ogc`), and as a WMS, WCS or WFS server (see

> section :ref:`label_qgisserver`). Additionally, you can publish your data on

Reviewer: ```suggestion
the :ref:`label_qgisserver`). Additionally, you can publish your data on
```

> section :ref:`working_with_ogc`), and as a WMS, WCS or WFS server (see
> the :ref:`QGIS-Server-manual`). Additionally, you can publish your data on
> the Internet using a webserver with QGIS Server, UMN MapServer or GeoServer installed.

Reviewer 2: ```suggestion
section :ref:`working_with_ogc`), and QGIS Server (see the :ref:`QGIS-Server-manual`)
allows you to publish your data through the WMS, WCS and WFS protocols on the Internet
using a webserver.
```
